llvm/test/CodeGen/AMDGPU/si-pre-allocate-wwm-regs.mir

```diff
@@ -0,0 +1,26 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -verify-machineinstrs -run-pass=si-pre-allocate-wwm-regs -o - -mcpu=tahiti %s  | FileCheck %s
+
+---
+
+name: pre_allocate_wwm_regs_strict
+tracksRegLiveness: true
+body: |
```

Comment:
This really needs checks for the MFI serialized allocated register. It is not yet serialized, but it really needs to be (and is currently causing issues for me)

Reply:
What exactly should be serialized for allocated registers?
I am working on serializing virtual register flags (currently there is only one WWM_REG)

Reply:
WWMSpills, SpillVGPRs, SpillPhysVGPRs, SGPRSpillsTo*